core-ui/src/components/Predefined/Details/EventSubscriptions/EventSubscriptions.details.js

```diff
@@ -0,0 +1,161 @@
+import { LayoutPanel } from 'fundamental-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  EMPTY_TEXT_PLACEHOLDER,
+  FormattedDatetime,
+  GenericList,
+} from 'react-shared';
+import { EventSubscriptionConditionStatus } from 'shared/components/EventSubscriptionConditionStatus';
+import { LayoutPanelRow } from 'shared/components/LayoutPanelRow/LayoutPanelRow';
+import './EventFilters.scss';
+import { Link } from 'fundamental-react';
+import { navigateToFixedPathResourceDetails } from 'react-shared';
+
+const EventSubscriptionConditions = eventSubscription => {
+  const { t, i18n } = useTranslation();
+
+  const conditions = eventSubscription?.status?.conditions;
+  const headerRenderer = _ => [
+    t('event-subscription.conditions.last-transition-time'),
+    t('event-subscription.conditions.reason'),
+    t('event-subscription.conditions.status'),
+    t('event-subscription.conditions.type'),
+  ];
+
+  const rowRenderer = condition => [
+    <FormattedDatetime
```

Comment:
Instead of the `FormattedDatetime` component, we should use the `ReadableCreationTimestamp` one here, as we use everywhere else (we use FormattedDatetime only to display the expiration date, which should be displayed differently).